index.html

```diff
@@ -6,14 +6,71 @@
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
     </head>
-    <body>
-        <header>
-            <nav>
-                
-            </nav>
+    <body class="body" id="body">
+        <header class="header">
+            <div class="container">
+                <div class="header_body">
+                    <a href="" class="header_logo">
```

Comment:
Please look up BEM practices for class naming, and then readjust your classes that way. ;)

Reply:
> Please look up BEM practices for class naming, and then readjust your classes that way. ;)

Ok, i will study it today and change the names

Reply:
Changed naming, but i'm little unsure about some places. However  in the future i will try to match the methodology